src/.env

```diff
@@ -0,0 +1 @@
+PORT=3008
```

Comment:
You don't want to check in a .env file. Typically this will have sensitive values you don't want in version control. You can set the PORT environment value in the npm run dev command if necessary. I would add .env to the.gitignore.

Reply:
Hi Rob please can you verify the jest test that I added with the mock data ?